pyqtgraph/parametertree/parameterTypes/basetypes.py

```diff
@@ -319,24 +325,26 @@ def __init__(self, param, depth):
         self.optsChanged(self.param, self.param.opts)
 
     def updateDepth(self, depth):
-        ## Change item's appearance based on its depth in the tree
-        ## This allows highest-level groups to be displayed more prominently.
-        if depth == 0:
-            for c in [0,1]:
-                self.setBackground(c, QtGui.QBrush(QtGui.QColor(100,100,100)))
-                self.setForeground(c, QtGui.QBrush(QtGui.QColor(220,220,255)))
-                font = self.font(c)
-                font.setBold(True)
-                font.setPointSize(font.pointSize()+1)
-                self.setFont(c, font)
-        else:
-            for c in [0,1]:
-                self.setBackground(c, QtGui.QBrush(QtGui.QColor(220,220,220)))
-                self.setForeground(c, QtGui.QBrush(QtGui.QColor(50,50,50)))
-                font = self.font(c)
-                font.setBold(True)
-                #font.setPointSize(font.pointSize()+1)
-                self.setFont(c, font)
+        """
+        Change set the item font to bold and increase the font size on outermost groups.
+        """
+        app = mkQApp()
+        palette = app.palette()
+        color = palette.base().color()
+        h, s, l, a = color.getHslF()
+        lightness = 0.5 + (l - 0.5) * .8
+        background = QtGui.QColor.fromHslF(h, s, lightness, a)
+        altBackground = color
+
+        for c in [0, 1]:
+            font = self.font(c)
+            font.setBold(True)
+            if depth == 0:
+                background = altBackground
+                font.setPointSize(self.fontPointSize() + 1)
+            self.setBackground(c, background)
```

Comment:
If you swap above, these need to swap, too :)

Reply:
Maybe just put 
``` python
 if depth == 0:
    font.setPointSize(self.fontPointSize() + 1)
    self.setBackground(c, background)
else:
    self.setBackground(c, altBackground)`
```
Then it should be clear what background goes where.

Reply:
Done.